qiling/os/windows/dlls/user32.py

```diff
@@ -341,12 +404,13 @@ def hook_GetOEMCP(ql, address, params):
 def hook_LoadStringA(ql, address, params):
     dst = params["lpBuffer"]
     max_len = params["cchBufferMax"]
+    # TODO fix the string
     string = "AAAABBBBCCCCDDDD" + "\x00"
```

Comment:
hardcoded by windows or we should allow user to change

Reply:
Technically no, should not be hardcoded nor the user should change it. Every string is described from a uID, but i'cant find the documentation of which string has which id

Reply:
maybe we profile it. that will be a better way